nerfstudio/configs/external_methods.py

```diff
@@ -192,6 +192,22 @@ class ExternalMethod:
     )
 )
 
+# Zip-NeRF
+external_methods.append(
+    ExternalMethod(
+        """[bold yellow]Zip-NeRF[/bold yellow]
+For more information visit https://docs.nerf.studio/nerfology/methods/zipnerf.html
+
+To enable Zip-NeRF, you must install it first by running:
+  [grey]pip install git+https://github.com/Jing1Ling/zipnerf-pytorch@support_nerfstudio#subdirectory=extensions/cuda 
```

Comment:
It looks like this link doesn't match the github link in the `zipnerf.md` file. Looks like this one points to your fork and the prior one points to the original repo.

Reply:
Sorry for missing the part in config file when updating Link. Now I've updated it.